hottbox/algorithms/decomposition/cpd.py

```diff
@@ -406,3 +410,194 @@ def _init_fmat(self, tensor, rank):
 
     def plot(self):
         print('At the moment, `plot()` is not implemented for the {}'.format(self.name))
+
+
+class Parafac2(BaseCPD):
```

Comment:
Make more emphasis on type of input to `decompose`, how it is different from other algorithms etc.

Reply:
Also, could you make changes to `__init__.py` so that this algothim would be available as such
```python
from hottbox.algorithms.decomposition import Parafac2
```